frontend-public/src/components/LandingPage.js

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import { Divider, Row, Col, Button } from "antd";
+
+export const LandingPage = () => (
+  <div>
+    <Row>
+      <Col xs={1} md={4} lg={4} />
+      <Col xs={22} md={16} lg={16}>
+        <h1 className="landing-page-title"> Welcome to MineSpace, an online portal for BC mines</h1>
+      </Col>
+      <Col xs={1} md={4} lg={4} />
+    </Row>
+    <Row>
+      <Col xs={1} md={4} lg={4} />
+      <Col xs={22} md={14} lg={10}>
+        <p>
+          The Mines Digital Service (MDS) is a digital product being iteratively developed and
+          designed to hold mine related data that is user-friendly, trustworthy and reliable. The
+          product will eventually replace the existing internal Mine Management System used by the
+          Ministry and be used to surface information to the{" "}
+          <a href="http://mines.nrs.gov.bc.ca/" target="_blank" rel="noopener noreferrer">
+            BC Mine Information website
+          </a>
+          . The MDS project was started in July to help inspectors manage the mines they work with
+          by replacing existing legacy systems and adding new functionality.
+        </p>
+        <br />
+        <p>
+          MineSpace is a pilot project as a part of the MDS digital product. MineSpace will be the
+          portal through which mines can interact with the Ministry. In time, Mines will be able to
+          submit documentation to the Ministry, as well as check the status of reporting
+          requirements, and documentation that they have submitted to the ministry for review. The
+          pilot project will run in the spring of 2019 with full integration in early 2020.
+        </p>
+      </Col>
+      <Col xs={0} md={2} lg={6} />
```

Comment:
What is the purpose of these empty columns?

Reply:
I need them to be there in lg and md to add white space around the content but not on the xs screen size.

Reply:
Okay, 

There is a better way to accomplish this than using empty elements, especially in this quantity. 
I can assist in this area if you'd like.